src/main/java/fr/alasdiablo/jerintegration/compat/miningmaster/MiningMasterWorldGen.java

```diff
@@ -0,0 +1,192 @@
+package fr.alasdiablo.jerintegration.compat.miningmaster;
+
+import fr.alasdiablo.jerintegration.api.WorldGenIntegration;
+import jeresources.api.IWorldGenRegistry;
+import jeresources.api.conditionals.Conditional;
+import jeresources.api.distributions.DistributionTriangular;
+import jeresources.api.drop.LootDrop;
+import jeresources.api.restrictions.DimensionRestriction;
+import jeresources.api.restrictions.Restriction;
+import net.minecraft.world.item.ItemStack;
+import org.infernalstudios.miningmaster.init.MMBlocks;
+import org.infernalstudios.miningmaster.init.MMItems;
+import org.jetbrains.annotations.NotNull;
+
+public class MiningMasterWorldGen extends WorldGenIntegration {
+    @Override
+    public void registerWorldGen(@NotNull final IWorldGenRegistry registry) {
+        registerStoneType(registry);
+        registerDeepslateType(registry);
+
+        registry.register(
+                new ItemStack(MMBlocks.HEART_RHODONITE_ORE.get()),
+                new DistributionTriangular(1, 1, 64, 64),
+                new Restriction(DimensionRestriction.NETHER),
+                true,
+                new LootDrop(new ItemStack(MMItems.HEART_RHODONITE.get()), 1, 1, Conditional.affectedByFortune)
+        );
+
+        registry.register(
+                new ItemStack(MMBlocks.KINETIC_OPAL_ORE.get()),
+                new DistributionTriangular(1, 1, 64, 64),
+                new Restriction(DimensionRestriction.NETHER),
+                true,
+                new LootDrop(new ItemStack(MMItems.KINETIC_OPAL.get()), 1, 1, Conditional.affectedByFortune)
+        );
+
+        registry.register(
+                new ItemStack(MMBlocks.POWER_PYRITE_ORE.get()),
+                new DistributionTriangular(1, 1, 64, 64),
+                new Restriction(DimensionRestriction.NETHER),
+                true,
+                new LootDrop(new ItemStack(MMItems.POWER_PYRITE.get()), 1, 1, Conditional.affectedByFortune)
+        );
+    }
+    private void registerStoneType(@NotNull final IWorldGenRegistry registry) {
```

Comment:
This function is call by `registerWorldGen`, i highly recommend to put it before `registerWorldGen`.